app/utils/CheckAction.scala

```diff
@@ -105,4 +106,23 @@ class Base58CheckAction(parser: BodyParsers.Default, address: String) extends Ac
 
 class Base58CheckActionFactory @Inject()(parser: BodyParsers.Default) {
   def apply(address: String): Base58CheckAction = new Base58CheckAction(parser, address)
+}
+
+class DateFromToCheckAction(parser: BodyParsers.Default, fromDate: String, toDate: String) extends ActionBuilderImpl(parser) {
+  private val sdf: SimpleDateFormat = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss")
```

Comment:
You have 2 exactly the same instances of SimpleDateFormat, maybe you should reuse them